### ISSUE TITLE
Pytorch Compatibility Bug with WebUI

### ISSUE THREAD

**Issue by RitchieAlpha** [microsoft/nni#1558]
**Short summary about the issue/question**:
The Problem is that i try the cifar10-pytorch example with pytorch 1.2 
However there can show only one Intermediate and can't show the final_result 
**Brief what process you are following**: 
![1](https://user-images.githubusercontent.com/38561576/65154114-7fa76000-da5d-11e9-96b0-826ad4977f66.PNG)
![2](https://user-images.githubusercontent.com/38561576/65154127-8635d780-da5d-11e9-84f9-ead739c56b9a.PNG)
![3](https://user-images.githubusercontent.com/38561576/65154383-03614c80-da5e-11e9-8d95-41515411cfe7.PNG)
![4](https://user-images.githubusercontent.com/38561576/65154390-05c3a680-da5e-11e9-80f8-f473f2015705.PNG)

**How to reproduce it**: 
 just follow [https://github.com/microsoft/nni/blob/master/docs/en_US/TrialExample/Cifar10Examples.md](url) and i use pytorch 1.2. I reduce the epoch to 5 and reduce the search space

**nni Environment**:
- nni version: v1.0
- nni mode(local|pai|remote):local
- OS: windows 10, 1903
- python version: 3.6
- is conda or virtualenv used?: conda  
- is running in docker?: no

**need to update document(yes/no)**:
no
**Anything else we need to know**:
Look forward to your reply!
Ritchie

**Comment by RitchieAlpha** [comment]
![6](https://user-images.githubusercontent.com/38561576/65159242-5b03b600-da66-11e9-941b-4194ddd4cca9.PNG)
![7](https://user-images.githubusercontent.com/38561576/65159243-5b03b600-da66-11e9-8a48-fa2753dc269b.PNG)
![8](https://user-images.githubusercontent.com/38561576/65159244-5b03b600-da66-11e9-9123-d6a8312ea4e9.PNG)

**Comment by RitchieAlpha** [comment]
I also tried mnist-pytorch ,you can see the second one

**Comment by lvybriage** [comment]
@RitchieAlpha 
Hi, RitchieAlpha. In this picture, we show the trial final result as the default metric rather than the latest intermediate result.
https://user-images.githubusercontent.com/38561576/65154114-7fa76000-da5d-11e9-96b0-826ad4977f66.PNG

**Comment by ultmaster** [comment]
@RitchieAlpha Any updates?

**Comment by cjissmart** [comment]
I have the same problem. When I use pytorch, the intermediate result noly updates once, which means only one intermediate result during one trial. And no final result is reported. The same code is run well on ubuntu. Besides, it's ok when I use tensorflow.
